Tests/test_backfield.py

```
@@ -8,14 +8,45 @@
+
         @dataclass
         class A:
             flag = False
-            b = Backfield(0, callback=lambda _: setattr(A, 'flag', True))
+            b = Backfield(0)
+
+        a = A()
+        A.b.bind(a, lambda _: setattr(A, 'flag', True))
+        a.b = 1
+
+        assert a.flag == True
```

Ruff (0.15.15)

[error] Avoid equality comparisons to `True`; use `a.flag:` for truth checks

Replace with `a.flag`

(E712)